examples/lowMach/lowMach.udf

```diff
@@ -9,13 +9,25 @@
 
 static occa::kernel userQKernel;
 static occa::kernel userVpKernel;
+static occa::kernel expansionCoeffKernel;
+occa::memory o_expansionCoeff;
 
 static dfloat P_DELTA;
 static dfloat P_GAMMA;
 
 void qtl(nrs_t* nrs, dfloat time, occa::memory o_div)
 {
+// ideal gas
   lowMach::qThermalIdealGasSingleComponent(time, o_div);
+
+// real gas
+  mesh_t *mesh = nrs->meshV;
+  platform_t* platform = platform_t::getInstance();  
+  cds_t* cds = nrs->cds;
+  o_expansionCoeff = platform->device.malloc(nrs->fieldOffset*sizeof(dfloat));
```

Comment:
Allocating o_expansionCoeff has to be done in UDF_Setup() otherwise you allocate every time you call qtl.

Reply:
ok